### ISSUE TITLE
Account Email Address Reset Password

### ISSUE THREAD

**Issue by trymeouteh** [voat/voat#934]
When you sign up to voat you do not need to give your email to join. But in the Manage Account page you can add your email address to your account? Does this allow me to reset my password if I forget it? And if not this would be a great feature to have.

**Comment by DanielFlamino** [comment]
Yes, email is optional upon registration and if set (during registration or in the settings page) can be used to recover the account in case you forget your password.
